torchci/lib/bot/mergeBot.ts

```diff
@@ -159,10 +161,14 @@ function mergeBot(app: Probot): void {
             option["classification"].replace(/['"]+/g, "")
           )
         ) {
+          const invalidClassificationMessage = option['classification'] != null ?
+          `(the classification you provided was: ${option['classification']})` :
+           "";
           await addComment(
             ctx,
-            "Revert unsuccessful: please retry the command and provide a classification, " +
-              `e.g. @pytorchbot revert -m="this breaks mac tests on trunk" -c="ghfirst".`
+            `Revert unsuccessful: please retry the command and provide a valid classification ${invalidClassificationMessage}, ` +
+            `\`e.g. @pytorchbot revert -m="this breaks mac tests on trunk" -c="ignoredsignal"\`.` +
```

Comment:
Maybe put this in a code fence on a separate line to avoid the confusion about punctuation